src/libs/ajax/methods/Methods.ts

```diff
@@ -31,6 +31,14 @@ export const Methods = (signal?: AbortSignal) => ({
     return res.json();
   },
 
+  methods: async (payload) => {
+    const res = await fetchOrchestration(
+      'api/methods',
+      _.mergeAll([authOpts(), jsonBody(payload), { signal, method: 'POST' }])
+    );
+    return res.json();
+  },
+
```

Comment:
Can this be moved to as `post` under the `method` block on L42? That block has get and other method related calls and `POST` would fit better there?

Reply:
I was trying to distinguish between the two different endpoints, as one includes query params and the other doesn't. But I can definitely move it down.

Reply:
I see. In that case it might be better to name it `postMethods` instead of `methods` as it will make it more clearer.